lms/djangoapps/ccx/migrations/0006_set_display_name_as_override.py

```diff
@@ -28,7 +29,11 @@ def save_display_name(apps, schema_editor):
 
     # Create `display_name` overrides for these CCX courses
     for ccx in ccx_list:
-        course = get_course_by_id(ccx.course_id, depth=None)
+        try:
+            course = get_course_by_id(ccx.course_id, depth=None)
+        except Http404:
+            log.error('Root course %s not found. Skipping saving course display name', ccx.course_id)
```

Comment:
@taranjeet It might be useful to mention the CCX that's being skipped here. I.e., do something like this:

```python
log.error(
    "Root course %s not found. Can't create `display_name` override for %s.",
    ccx.course_id,
    ccx.display_name
)
```